tasks/main.yaml

```diff
@@ -20,7 +21,8 @@
 - name: Get Tuned recommended profile
   shell: tuned-adm recommend 2> /dev/null || echo -n " *fail"
   register: recommend
-
+  check_mode: no
```

Comment:
see above

Reply:
tested
same issue

TASK [linux-system-roles.tuned : set_fact] **************************************************************
fatal: [workstation]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'stdout'\n\nThe error appears to be in '/home/student/.ansible/roles/linux-system-roles.tuned/tasks/main.yaml': line 13, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- set_fact:\n  ^ here\n"}

Reply:
tested with `changed_when: false` in addition to `check_mode: no`
it is better